src/VisualStudio/Setup/codeAnalysisService64.servicehub.service.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "host": "desktopClr",
+  "hostId": "RoslynCodeAnalysisService",
```

Comment:
❓ Does this already exist?

Reply:
what do you mean? this is service hub config file that I added for AnyCpu service hub host.

are you talking about file, or host id? these are new files I added.

Reply:
Currently we have this file getting installed with Visual Studio 2017:

> C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\ServiceHub\Hosts\ServiceHub.Host.CLR.x86\ServiceHub.RoslynCodeAnalysisService32.exe

But we don't have this:

> C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\ServiceHub\Hosts\ServiceHub.Host.CLR.x64\ServiceHub.RoslynCodeAnalysisService.exe

Based on the `hostId` value here and comparing with the `hostId` in the original 32-bit OOP files, it appears the new executable will need to come from somewhere, but I wasn't sure where/how. I was asking if you already took steps, perhaps in another repository, to create the new host executable so it can be installed.

Reply:
hostid has nothing to do with exe. service hub team create those exe to distinguish each team's usage of service hub, but it is cosmetic, not related to functionality. I asked them to create one for us for AnyCpu. but regardless of them doing it, functionality wise, things work fine.